requirements.in

```diff
@@ -1,4 +1,5 @@
 flask-exceptions
 jinja2>=2.10.1
-marshmallow==3.0.0b13
+marshmallow==3.1.1
 protobuf
+six==1.11.0
```

Comment:
Didn't this need to be `1.12.0`?

Reply:
This was the only way I could get the dependencies to play nice and it works 🤷‍♂

Reply:
👍